src/main/java/com/gluonhq/substrate/util/ProcessRunner.java

```diff
@@ -301,7 +316,13 @@ public static Integer executeWithFeedback(String name, String... args) throws IO
 
     private Process setupProcess(String processName, File directory) throws IOException {
         ProcessBuilder pb = new ProcessBuilder(args);
-        Logger.logDebug("PB Command for " +  processName + ": " + String.join(" ", pb.command()));
+        String join = pb.command().stream().map(s -> {
+            if (passwords.contains(s)) {
+                return s.substring(0, 1) + "******";
```

Comment:
There should be a check for empty passwords. In any case, can't we even skip printing the first character of passwords?

Reply:
hmm, it doesn't make much sense to call `runner.setSecretArgs("")`, but that will cause an exception. I'll fix it.

Reply:
In fact, the issue affects the regular `setArg("")` call, as that can lead to errors when processing the command, but this is probably for another issue.

Reply:
OK. If you spotted another issue, please file it before it gets forgotten.